### PR TITLE
Added Hindi Chars

### PR DESCRIPTION
**Pull request by Akshat-mehrotra**

### CODE REVIEW

**Comment by aviks** [comment]
Cool, thanks. but मात्रा ?

File: src/text.jl [hunk 1]
```diff
@@ -269,6 +269,40 @@ font = Dict(
 'š' => Bool.([0 1 0 1 0 ; 0 0 1 0 0 ; 0 0 0 0 0 ; 0 1 1 1 1 ; 1 0 0 0 0 ; 0 1 1 1 0 ; 0 0 0 0 1 ; 1 1 1 1 0 ; ]),
 'Ž' => Bool.([0 1 0 1 0 ; 0 0 1 0 0 ; 0 0 0 0 0 ; 1 1 1 1 1 ; 0 0 0 1 0 ; 0 0 1 0 0 ; 0 1 0 0 0 ; 1 1 1 1 1 ; ]),
 'ž' => Bool.([0 1 0 1 0 ; 0 0 1 0 0 ; 0 0 0 0 0 ; 1 1 1 1 1 ; 0 0 0 1 0 ; 0 0 1 0 0 ; 0 1 0 0 0 ; 1 1 1 1 1 ; ]),
+# Hindi
```
Comment (aviks) [inline src/text.jl]: Devnagari. Hindi/Marathi/Nepali can be written in the devnagari script.

**Comment by Akshat-mehrotra** [comment]
If i add matras, then I would have to add every maatra to every character, that would be total of 208 characters!! What do you advise I do?

**Comment by aviks** [comment]
No no, matras are individual unicode characters by themselves. See https://en.wikipedia.org/wiki/Devanagari_(Unicode_block)

There is one problem -- the spacing and positioning of the matras is complex, but that is a problem for later. For now, let's get the matra shapes into the system.

**Comment by Akshat-mehrotra** [comment]
Oh ok then, I thought that I'd have to add every matra to evey character. I
add individual matras as characters!

File: src/text.jl [hunk 2]
```diff
@@ -269,6 +269,40 @@ font = Dict(
...
+'क' => Bool.([1 1 1 1 1;  0 0 1 0 0; 1 1 1 1 1; 1 0 1 0 1; 1 0 1 0 1; 1 1 1 0 1; 0 0 1 0 0; 0 0 1 0 0]),
+'ख' => Bool.([1 1 1 1 1; 0 1 0 0 1; 0 1 1 1 1; 1 0 1 0 1; 1 0 1 1 1; 0 1 0 0 1; 0 0 1 0 1; 0 0 0 0 0 ]),
+'ग' => Bool.([1 1 1 1 1; 0 1 0 1 0; 0 1 0 1 0; 1 1 0 1 0; 1 1 0 1 0; 0 0 0 1 0; 0 0 0 1 0; 0 0 0 1 0; ]),
+'घ' => Bool.([1 1 1 1 1; 1 0 0 1 0; 1 0 0 1 0; 0 1 0 1 0; 1 0 0 1 0; 1 1 1 1 0; 0 0 0 1 0; 0 0 0 1 0; ]),
+'ङ' => Bool.([1 1 1 1 1; 0 0 0 1 0; 0 1 1 1 0; 0 1 0 0 1; 0 1 1 1 0; 0 0 0 1 0; 0 1 1 1 0; 0 0 0 0 0; ]),
+'च' => Bool.([1 1 1 1 1; 0 0 0 1 0; 0 0 0 1 0; 1 1 0 1 0; 0 1 0 1 0; 0 0 1 1 0; 0 0 0 1 0; 0 0 0 1 0; ]),
+'छ' => Bool.([1 1 1 1 0; 1 0 0 0 0; 1 0 0 0 0; 0 1 0 0 0; 1 0 0 1 1; 1 0 0 1 0; 0 1 1 1 0; 0 0 0 0 0; ]),
+'ज' =>  Bool.([1 1 1 1 1; 0 0 0 0 1; 0 0 0 0 1; 1 0 1 1 1; 1 0 1 0 1; 1 1 1 0 1; 0 0 0 0 1; 0 0 0 0 1; ]),
```
Comment (kimttfung) [inline src/text.jl]: The extra space here might not make it work :)
```suggestion
'ज' => Bool.([1 1 1 1 1; 0 0 0 0 1; 0 0 0 0 1; 1 0 1 1 1; 1 0 1 0 1; 1 1 1 0 1; 0 0 0 0 1; 0 0 0 0 1; ]),
```

**Comment by Akshat-mehrotra** [comment]
I'm making all the matars of the normal size i.e. I am not shrinking them to their perspective size right now, because doing that would make most of them indistinguishable. Edit: I am making them as tiny as possible to be as close to the real perspective size as possible

**Comment by Akshat-mehrotra** [comment]
added all matars

**Pull request merged**